tests/console/wicked_before_test.pm

```diff
@@ -13,13 +13,18 @@
 use base "consoletest";
 use strict;
 use testapi;
-use utils qw(systemctl setup_static_network);
+use utils qw(systemctl setup_static_network is_sle sle_version_at_least);
 use mm_network;
 
 sub run {
     my ($self) = @_;
     select_console('root-console');
-    assert_script_run "rcSuSEfirewall2 stop";
+    if (is_sle() && sle_version_at_least('15')) {
+        systemctl('stop firewalld');
+    }
+    else {
+        assert_script_run('rcSuSEfirewall2 stop');
+    }
```

Comment:
The commands are actually equivalent, it's only a different name for the service so the following can also work

```perl
my $service = (is_sle && sle_version_at_least('15')) ? 'firewalld' : 'SuSEfirewall2';
systemctl "stop $service";
```